components/activity_and_insights/activity_and_insights_link/insights_tour_tip/insights_tour_tip.tsx

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {memo, useCallback, useEffect, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {setInsightsInitialisationState} from 'mattermost-redux/actions/preferences';
+
+import {getBool} from 'mattermost-redux/selectors/entities/preferences';
+
+import {GlobalState as EntitiesGlobalState} from 'mattermost-redux/types/store';
+
+import {Preferences} from 'mattermost-redux/constants';
+
+import {showInsightsPulsatingDot} from 'selectors/insights';
+
+import {GlobalState} from 'types/store';
+
+import insightsPreview from 'images/Insights-Preview-Image.jpg';
+
+import TourTip, {useMeasurePunchouts} from 'components/widgets/tour_tip';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
+import {OnboardingTaskCategory, OnboardingTaskList} from 'components/onboarding_tasks';
+
+const title = (
+    <FormattedMessage
+        id='activityAndInsights.tutorialTip.title'
+        defaultMessage='Introducing: Insights'
+    />
+);
+
+const screen = (
+    <>
+        <FormattedMarkdownMessage
+            id='activityAndInsights.tutorialTip.description'
+            defaultMessage='Check out the new Insights feature added to your workspace. See what content is most active, and learn how you and your teammates are using your workspace.'
+        />
+        <img
+            src={insightsPreview}
+            className='insights-img'
+        />
+    </>
+
+);
+
+const prevBtn = (
+    <FormattedMessage
+        id='activityAndInsights.tutorial_tip.notNow'
+        defaultMessage='Not now'
+    />
+);
+
+const nextBtn = (
+    <FormattedMessage
+        id='activityAndInsights.tutorial_tip.viewInsights'
+        defaultMessage='View insights'
+    />
+);
+
+const InsightsTourTip = () => {
+    const dispatch = useDispatch();
+    const showTip = useSelector(showInsightsPulsatingDot);
+    const showNextStepsEphemeral = useSelector((state: GlobalState) => state.views.nextSteps.show);
```

Comment:
all the validations related to nextSteps are not longer relevant. Also, the nextSteps was removed from the state.

Reply:
The check onboarding function only contains one thing  now:
```
const isOnboardingOngoing = useCallback(() => {
        if (showTaskList) {
            return true;
        }
        return false;
}, [showTaskList]);
```